index.html

```diff
@@ -1,152 +1,115 @@
-            alt="makeup-image"
-          />
-        </div>
-        <div class="category-list-item">
-          <div class="category-list-item-name">Haircare</div>
-          <img
-            class="category-list-item-image"
-            src="/assets/hair.jpg"
-            alt="makeup-image"
-          />
-        </div>
-        <div class="category-list-item">
-          <div class="category-list-item-name">Fragrances</div>
-          <img
-            class="category-list-item-image"
-            src="/assets/Fragrances.jpg"
-            alt="makeup-image"
-          />
-        </div>
+  <div class="categories">
+    <div class="categories-title">Top Categories</div>
+    <div class="category-list">
+      <div class="category-list-item">
+        <div class="category-list-item-name">Makeup</div>
+        <img class="category-list-item-image" src="/assets/makeup.jpg" alt="makeup-image" />
+      </div>
+      <div class="category-list-item">
+        <div class="category-list-item-name">Skincare</div>
+        <img class="category-list-item-image" src="/assets/skin.jpg" alt="makeup-image" />
+      </div>
+      <div class="category-list-item">
+        <div class="category-list-item-name">Haircare</div>
+        <img class="category-list-item-image" src="/assets/hair.jpg" alt="makeup-image" />
+      </div>
+      <div class="category-list-item">
+        <div class="category-list-item-name">Fragrances</div>
+        <img class="category-list-item-image" src="/assets/Fragrances.jpg" alt="makeup-image" />
       </div>
     </div>
+  </div>
 
-    <div class="onlyat-section">
-      <div class="onlyat-section-title">Only At Stunners Hub</div>
-      <div class="onlyat-list">
-        <img class="onlyat-item" src="/assets/collection1.jfif" alt="" />
-        <img class="onlyat-item" src="/assets/collection2.jfif" alt="" />
-      </div>
+  <div class="onlyat-section">
+    <div class="onlyat-section-title">Only At Stunners Hub</div>
+    <div class="onlyat-list">
+      <img class="onlyat-item" src="/assets/collection1.jfif" alt="" />
+      <img class="onlyat-item" src="/assets/collection2.jfif" alt="" />
     </div>
+  </div>
 
-    <div class="top-brands">
-      <div class="top-brands-title">Top Brands</div>
-      <div class="top-brands-list">
-        <img class="top-brands-item" src="/assets/brand1.jpg" alt="" />
-        <img class="top-brands-item" src="/assets/brand2.jpg" alt="" />
-        <img class="top-brands-item" src="/assets/brand3.jpg" alt="" />
-        <img class="top-brands-item" src="/assets/brand4.jpg" alt="" />
-      </div>
+  <div class="top-brands">
+    <div class="top-brands-title">Top Brands</div>
+    <div class="top-brands-list">
+      <img class="top-brands-item" src="/assets/brand1.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand2.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand3.jpg" alt="" />
+      <img class="top-brands-item" src="/assets/brand4.jpg" alt="" />
```

Comment:
Please provide alt description for accessibility

Reply:
done